package.json

```diff
@@ -54,5 +54,9 @@
   "bugs": {
     "url": "https://github.com/noahsalvi/svelte-notion/issues"
   },
-  "homepage": "https://github.com/noahsalvi/svelte-notion#readme"
+  "homepage": "https://github.com/noahsalvi/svelte-notion#readme",
+  "dependencies": {
```

Comment:
Do we need those as dependencies or can they be included as just dev dependencies?

Reply:
Opps, yep `@rollup/plugin-commonjs` should be a dev dependency. `prismjs` needs to be a dependency to highlight code dynamically

Reply:
👍